frontend/lib/i18n/locales/nl/election.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "Verkiezing",
+  "name_plural": "Verkiezingen",
```

Comment:
"name" makes me think this might be the name of the election, e.g. "Gemeenteraadsverkiezingen 2026". Hard to come up with a better term, maybe just "election" and "elections"?

I als noticed that in `polling_station.json` we have "title": "Stembureau". So is `election.json` a generic election translation file and `polling_station.json` specific to the `/data-entry` page? If that's the case I think we need to make that distinction clearer.

The use of generic terms also makes me wonder if we need a function to capitalize words so we can define "verkiezing" once and use the function for titles etc.

Maybe all of this should be a new issue. Because improving the structure of the files is a lot easier after the hard work of adding everything has been done.

Reply:
there was an idea floating around to have a T (instead of 't') to capitalize the translation.
or just add some flags.

Reply:
@lkleuver @marlonbaeten Should we add an issue to this epic to:
1. Refactor the structure of the translation files if/where needed?
2. Add a short README to describe that structure?

Reply:
I'd like to scope this to "just" providing all translation keys and setup a discussion/issue/documentation for how to best structure translations

Reply:
created issue #686